src/api/v2/ResourceGroupsExtensionManager.ts

```diff
@@ -27,4 +40,37 @@ export class ResourceGroupsExtensionManager {
             }
         }
     }
+
+    async activateApplicationResourceProviders(): Promise<void> {
```

Comment:
Shouldn't this be to activate branches, not providers? (Since only RG has the single app resource provider)

Reply:
It's to activate all extensions that contribute resources immediately prior to querying for resources; it's true that RG is the only extension today that registers a provider, but it may not be the only one forever.  (The API allows for others.)